### PR TITLE
Update fields.rst

### PR DESCRIPTION
**Pull request by lindsayad**
In description of gradient, change "divergence of a vector field" to "divergence of a scalar field"

### CODE REVIEW

File: doc/src/modules/physics/vector/fields.rst [hunk 1]
```diff
@@ -229,7 +229,7 @@ denoted by :math:`\nabla f` is given by -
 \frac{\partial f}{\partial y}  \mathbf{\hat{j}} +
 \frac{\partial f}{\partial z} \mathbf{\hat{k}}`
 
-To compute the divergence of a vector field in :mod:`physics.vector`, you
+To compute the divergence of a scalar field in :mod:`physics.vector`, you
```
Comment (cbm755) [inline doc/src/modules/physics/vector/fields.rst]: I think this should say "gradient of a scalar field".  And a few lines above on ln 225 as well.

**Comment by cbm755** [comment]
Thanks for this.

I'm -1 for this change as is, but please fix up and I'll be happy to merge this.

**Comment by lindsayad** [comment]
I agree. Made suggested changes

**Comment by cbm755** [comment]
Ok great!  failed test looks like unrelated timeout, so merging.

**Comment by cbm755** [comment]
+1, will merge in 24 hours or so.

**Comment by cbm755** [comment]
@lindsayad please +1 #9393, or let me know if you want different email, etc.

**Pull request merged**